services/wikiClient.js

```diff
@@ -0,0 +1,42 @@
+const axios = require('axios');
+const Url = require('url');
+
+const defaultParams = { action: 'query', format: 'json' };
+const apiPath = 'w/api.php';
+
+function transform(data) {
+  const { query } = data;
+  return query;
+}
+
+class WikiClient {
+  constructor() {
+    this.client = axios.create({
+      headers: {
+        common: {
+          'Content-Type': 'application/json; charset=utf-8',
+          Accept: 'application/json',
+        },
+      },
+      timeout: 5000,
+    });
+  }
+
+  getResultsFromApi(titles, prop, wikiUrl, params = {}) {
```

Comment:
Looking at this now, I wonder if we should rather pass an object 🤔. Four params may already be above what can be done sane when using positional args only 🤔

Reply:
Personally, as we only use this `getResultsFromApi` function internally I find it okay to keep it like it is.